FabNESO/tasks.py

```diff
@@ -99,9 +185,19 @@ def neso_ensemble(
         solver: Which NESO solver to use.
         conditions_file_name: Name of conditions XML file in configuration directory.
         mesh_file_name: Name of mesh XML in configuration directory.
+        processes: Number of processes to run each job in ensemble on.
```

Comment:
Similarly this sounds a bit weird.
```suggestion
        processes: Number of processes to run in each job in the ensemble.
```